CanonicalLtd/documentation-builder
# Add documentation

That can also be built with the documentation builder.

Also a bunch of drive-by fixes.

Fixes #13

QA
--

Pull down this branch and run `documentation-builder` in this directory.

Check the built documentation.
